SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -82,6 +82,16 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.imageViews.map {$0.contentMode}, [.center, .center, .center])
     }
 
+    func testImageViewsIsAccessibilityElement() {
+        let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
+        XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Comment:
From my experience test files often tend to be quite bigger than files of the types they test.

I think that bigger tests, even if they are simple, also add some cognitive overhead if compared to smaller tests which are more concise (but not overly _clever_). With smaller tests, you can navigate more quickly and see more tests per each screen, so I think conciseness is a good thing.

- `map` is concise and in some cases it lets us avoid unnecessary `guard` statement.
- AFAIK `map` is not being considered _clever coding_ by Swift community

So I would vote for using simple `map`s and `flatMap`s in tests.